kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm-search/src/test/resources/userinfo.properties

```diff
@@ -0,0 +1,4 @@
+Administrator=Administrator@domain.com:en-UK:Administrator
```

Comment:
This file is used for resolving of email address, I guess you8 don't need it for your tests. If so please delete it.